src/Components/SearchBar.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+var text = "practise";
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <div className="SearchBar">
```

Comment:
Do you think your search bar would be inside your header section.
Or do you think your header section would be inside your search bar?